### DISCUSSION TITLE
Elsa 3.0 - Webhook UI

### DISCUSSION THREAD

**Discussion by schneidersympatient** [elsa-workflows/elsa-core#4344] [Q&A] [answered]
Hi there,
is there any UI available for consuming webhooks in Elsa 3.0?

I've only found https://github.com/elsa-workflows/elsa-core/issues/3592 which mentions that an UI should be provided, however after following the documentation to install Elsa Server and Studio, as well as adding the Webhooks module to my services, the UI does not provide anything to add webhooks.

**Comment by sfmskywalker** [comment, marked as answer]
Hi @schneidersympatient,

Thank you for reaching out. You're correct, as of now, the UI for consuming webhooks in Elsa 3.0 has not been added. The Webhooks Studio module you're seeing is currently just a scaffold, similar to the Security and Secrets modules. I do intend on working on it at some point, but its development largely depends on other priorities.

Thanks for your understanding and patience.
